### ISSUE TITLE
ocrmypdf --version crashes with pikepdf 2.2.1 

### ISSUE THREAD

**Issue by so-o** [jbarlow83/OCRmyPDF#698]
Here is the core dump:

```
Program received signal SIGSEGV, Segmentation fault.
0x00007ffff6ddb51f in pybind11::detail::type_caster_base<QPDFObjectHandle>::make_move_constructor<QPDFObjectHandle, void>(QPDFObjectHandle const*)::{lambda(void const*)#1}::_FUN(void const*) ()
   from .../OCRmyPDF/lib/python3.8/site-packages/pikepdf/_qpdf.cpython-38-x86_64-linux-gnu.so
```
   
OCRmyPDF 11.4.0
pikepdf 2.2.1
qpdf 10.0.4
python 3.8.5

Could be an issue with pikepdf.

**Comment by jbarlow83** [comment]
Can you provide some additional information, in particular what OS is this, and what commands did you use to install it?

**Comment by so-o** [comment]
Ubuntu 20.04 LTS. OCRmyPDF, qpdf and pikepdf 2.2.1 were installed from the deposit on github.

**Comment by jbarlow83** [comment]
I pushed pikepdf 2.2.1 to pypi. Please try that instead. The binary wheels provide their own version of libqpdf and should bypass the issue you encountered. 

There are no C++ code changes between 2.2.0 and 2.2.1, so the difference is something to do with custom compilation of  qpdf and pikepdf.

The command `pip3 install pikepdf` will not automatically find 10.0.4 on its own without some explicit custom steps and probably linked against whatever qpdf ships with Ubuntu 20.04. I'm not sure why it linked at all.

**Comment by so-o** [comment]
Installing pikepdf 2.2.1 from the wheel on pypi works, and the PDF/A-1B generated by OCRmyPDF is valid for veraPDF. Thanks!

The version of qpdf installed on Ubuntu 20.04 is 9.1.1. I did install in /usr/local/lib version 10.0.4 of qpdf compiled from the git deposit. The directory _/usr/local/lib_ is listed in _/etc/ld.so.conf.d/libc.conf_. I did a `sudo ldconfig` before installing pikepdf from the source tree. I can't uninstall the official qpdf, many programs depend on it. Why isn't the linker using the libraries in _/usr/local/lib_?
